ebi_eva_common_pyutils/ncbi_utils.py

```diff
@@ -78,3 +80,17 @@ def get_ncbi_assembly_name_from_term(term):
         raise ValueError(f'Cannot resolve assembly name for assembly {term} in NCBI. '
                          f'Found {",".join([str(a) for a in assembly_names])}')
     return assembly_names.pop() if assembly_names else None
+
+
+def retrieve_species_names_from_tax_id_ncbi(taxid):
```

Comment:
```suggestion
def retrieve_species_scientific_name_from_tax_id_ncbi(taxid):
```

Reply:
Somewhat unrelated but I've been using [ete3](https://pypi.org/project/ete3/) recently for this kind of operation:
```python
from ete3 import NCBITaxa
taxid = 9606
ncbi= NCBITaxa()
rank = taxa.get_rank([taxid])[taxid]
if rank not in ['species', 'subspecies']:
    logger.warning('Taxonomy id %s does not point to a species', taxid)
else:
    return ncbi.get_taxid_translator([taxid])[taxid]
```